app/modules/fcm/controllers/cytoscape.js

```diff
@@ -1187,9 +1215,19 @@ angular.module('pcApp.fcm.controllers.cytoscapes', [])
     }) // end AssociationController
 
     .controller('ModelController', function ($scope, $modalInstance, data, FCMModelsDetail) {
-        $scope.user = [
-            { FCMModelId: -1 }, { title: '' }, { description: '' }, { keywords: '' }
-        ];
+        if ($.isEmptyObject(data)) {
+            $scope.user = [
+                { FCMModelId: -1 }, { title: '' }, { description: '' }, { keywords: '' }
+            ];
+        }
+        else {
+            $scope.user = {
```

Comment:
@mohammedbahja as @FabiApfelkern mentioned, that's either a wrong variable name or wrong data. 
Also within the two cases, the user variable has two different data types. in the first case it's an array of objects where on the second case, it's a single object with all the individual properties of the array mentioned before.
To sum it up, this is definitely not a user variable, based on the data you store, and it's completely wrong to have different types stored on the same variable name. You should also fix the code that is using this.
Either way, i'm closing this PR.